frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -69,7 +66,8 @@ const renderTable = (
 };
 
 const renderAllTeamsTable = (
-  allTeamsScheduledQueriesList: IGlobalScheduledQuery[],
+  // allTeamsScheduledQueriesList: IEditScheduledQuery[],
```

Comment:
Remove commented line?

Reply:
@lukeheath Thanks for catching this! This line is now type IScheduledQuery[], I was having my types not match and forgot to update this!